src/Installers/Windows/Wix.targets

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup Condition="'$(OutputType)' == 'package'">
     <InstallDir>$(ProductName)</InstallDir>
     <DepProviderKey>Microsoft.$(ProductNameShort)_$(Platform)_$(Lang),v$(PackageVersion)</DepProviderKey>
-    <DefineConstants>$(DefineConstants);DepProviderKey=$(DepProviderKey)</DefineConstants>
+    <!-- <DefineConstants>$(DefineConstants);DepProviderKey=$(DepProviderKey)</DefineConstants> -->
```

Comment:
Please undo before merging. This is important for the other installers in this repo.